app/controllers/Books.js

```diff
@@ -0,0 +1,58 @@
+const db = require('../../database/models')
+const responses = require('../helpers/responses')
+
+class Books {
+  static async GetAllBooks(req, res) {
+    try {
+      const results = db.Book.findAll();
+      const message = [200, 'Books retrieved successfully', true];
+      responses.response(res, message, results);
+    } catch (error) {
+      responses.handleError(error.toString(), 500, res);
+    }
+  }
+
+  static async GetOneBook(req, res) {
+    try {
+      const results = await db.Book.findOne({
+        where: { id: req.params.id },
+        include: [
+          {
+            model: db.User,
+            attributes: ['firstName', 'lastName'],
+            as: 'Authors',
+            through: { attributes: [] },
+          },
+        ],
+      });
+      const message = [200, 'Book retrieved successfully', true];
+      responses.response(res, message, results);
+    } catch (error) {
+      responses.handleError(error.toString(), 500, res);
+    }
+  }
+
+  static async CreateBook(req, res) {
+    try {
+      const results = db.Book.create({
+        title: req.body.title,
+      })
+      const message = [201, 'Book created successfully', true];
+      responses.response(res, message, results);
+    } catch (error) {
+      responses.handleError(error.toString(), 500, res);
+    }
+  }
+
+  static async EditBook(req, res) {
+    try {
+      const results = await db.Book.findById(req.params.id);
```

Comment:
this implementation does not really do what the method name says, `editing a book`

Reply:
Worked on it.